web/src/components/onboarding/IndustryDropdown.test.tsx

```diff
@@ -10,4 +11,40 @@ describe("Industry Dropdown", () => {
 
     expect(within(items[0]).getByTestId("generic")).toBeInTheDocument();
   });
+
+  it("displays the generic industry as the single option when there is no industry match", async () => {
+    render(<IndustryDropdown />);
+    const searchTerm = `some-industry-${randomInt()}`;
+
+    fireEvent.click(screen.getByLabelText("Industry"));
+    fireEvent.change(screen.getByLabelText("Industry"), {
+      target: { value: searchTerm },
+    });
+
+    fireEvent.click(screen.getByLabelText("Industry"));
+    expect(screen.getByTestId("generic")).toBeInTheDocument();
+    expect(screen.getAllByText(searchTerm).length).toEqual(1);
+    expect(screen.queryByTestId("certified-public-accountant")).not.toBeInTheDocument();
+  });
+
+  it("updates the value of the industry dropdown input when the no-matching industries button is clicked", () => {
+    render(<IndustryDropdown />);
+
+    const inputElement = screen.getByLabelText("Industry");
+    fireEvent.click(inputElement);
+    fireEvent.change(inputElement, { target: { value: "ac5" } });
+    fireEvent.click(screen.getByTestId("generic"));
+
+    expect((screen.getByLabelText("Industry") as HTMLInputElement).value).toBe("All Other Businesses");
+  });
```

Comment:
I'm not sure this test is adding value, because it's testing the intrinisic behavior of the dropdown. Part of our testing philosophy is avoiding testing the "internals" of a component that we use from a third-party library like Material UI. We can test their interactions with our code, and we test any configuration we do to them. 

But (if I'm reading this test correctly), this will always pass because it's expecting the value of the input field to change to the option we click, when we click an option. I'm pretty sure that's the default behavior for this MUI component, and there's no world in which this test would fail?

Again, I could be missing something, let me know if there's something else you were intending here. But if it's just testing the MUI component behavior, I think we can delete this test altogether.